org.eclipse.xtext.xbase/src/org/eclipse/xtext/xbase/annotations/validation/UnresolvedFeatureCallTypeAwareMessageProvider.java

```diff
@@ -136,7 +136,8 @@ protected boolean isStaticMemberCallTarget(EObject contextObject) {
 	}
 
 	/**
-	 * @Nullable
+	 * Returns a part of the message that describes the nature of the requested object, including a leading space.
+	 * The result is either empty or of the shape {@code " to <article> <type>"}.
 	 */
 	protected String getTypeName(EClass c, EStructuralFeature referingFeature) {
```

Comment:
when you are on it: rename referingFeature -> referringFeature

Reply:
did a search on the code base, there are some 100s more hits for "refering". But if I'm correct here, it must be "referring". If you agree that this is a recurring typo, I'll open a separate issue.